sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/KnownMethods.cs

```diff
@@ -22,6 +22,7 @@
 using System.Linq;
 using System.Runtime.Serialization;
 using Microsoft.CodeAnalysis;
+using System.Diagnostics;

```

Comment:
It is not perfect if the usings are not sorted. And also if someone (who has a kb shortcut) sorts them later, it looks like a weird change...

There is a VS setting for usings order - System.* should be first, then others. I have a custom kb shortcut configured for remove and sort and it became like a second nature before saving to press Ctrl+K, Ctrl+D (format), then Ctrl+K, Ctrl+O (custom remove and sort usings)...